### DISCUSSION TITLE
Upgraded to version 1.54.25.0 which doesn't exist in Releases/Tags

### DISCUSSION THREAD

**Discussion by guentherwieser** [bassmaster187/TeslaLogger#1101] [Q&A]
Just checking out of curiosity and to avoid and malicious attacks, my system just updated to 1.54.25.0 (via the docker way of life), but this release is not described here in Github, nor does this tag exist. Any hints how this can happen? It also contiuously shows the banner "Bitte aktualisieren Sie auf die neueste docker-compose.yml-Datei." even though there is nothing newer (presumably).

Any ideas? Thx!

Günther

**Comment by bassmaster187** [comment]
Sorry, my bad. I forgot to update the release tag. 
What version of grafana are you running?

**Reply by guentherwieser** [reply]
Ah, thx! No problem, happy that all is as expected.

Grafana version is 10.0.1

**Reply by guentherwieser** [reply]
So the banner is gone now, maybe it was related to the missing release tag?
